src/components/sections/forcedslots/ExoticSlots.jsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Checkbox,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@mui/material';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { makeStyles } from 'tss-react/mui';
+import { getForcedSlots } from '../../../state/slices/forcedSlots';
+import { changeExotic, getExoticsData, getPriority } from '../../../state/slices/priorities';
+import { GEAR_SLOTS, WeaponTypes } from '../../../utils/gw2-data';
+
+const useStyles = makeStyles()((theme) => ({
+  tableCollapse: {
+    borderCollapse: 'collapse !important',
+    marginBottom: '0px !important',
+  },
+  checkbox: {
+    padding: '6px',
+  },
+}));
+
+const ExoticSlots = () => {
+  const { classes } = useStyles();
+
+  const dispatch = useDispatch();
+  const forcedSlots = useSelector(getForcedSlots);
+  const dualWielded = useSelector(getPriority('weaponType'));
+  const affixes = useSelector(getPriority('affixes'));
+  const exotics = useSelector(getExoticsData);
+
+  let SLOTS = GEAR_SLOTS;
+  if (dualWielded !== WeaponTypes.dualWield) {
+    SLOTS = GEAR_SLOTS.slice(0, 13);
+  }
+
+  const handleChange = (index, affix) => (event) => {
+    dispatch(changeExotic({ index, affix, value: event.target.checked }));
+  };
+
+  return (
+    <TableContainer>
+      <Table className={classes.tableCollapse}>
+        <TableHead>
+          <TableCell padding="none" />
+          {SLOTS.map((slot, index) => (
+            // eslint-disable-next-line react/no-array-index-key
+            <TableCell padding="none" key={index}>
+              {slot.short}
+            </TableCell>
+          ))}
+        </TableHead>
+        <TableBody>
+          {affixes.map((affix) => (
+            <TableRow key={affix}>
+              <TableCell padding="none">{affix}</TableCell>
+              {SLOTS.map((slot, index) => (
+                // eslint-disable-next-line react/no-array-index-key
+                <TableCell padding="none" sx={{ padding: 0.3 }} key={index}>
+                  <Checkbox
+                    size="small"
+                    classes={{ root: classes.checkbox }}
+                    checked={Boolean(exotics?.[affix]?.[index])}
+                    disabled={forcedSlots[index] !== null}
```

Comment:
These checkboxes can be disabled when their corresponding per-slot exclusion is selected rather than according to forced slots!

(actually, forced slots being separate here becomes sort of a mess, but that's kind of on me :D)